src/components/views/rooms/MessageComposerButtons.tsx

```diff
@@ -74,7 +75,7 @@ const MessageComposerButtons: React.FC<IProps> = (props: IProps) => {
         ];
         moreButtons = [
             uploadButton(), // props passed via UploadButtonContext
-            showStickersButton(props),
+            showStickersButton(props, room),
```

Comment:
Would it make sense for it to be a `prop` similar to `props.showPollsButton`?

Reply:
Thinking of it I would change it this way :+1:

Reply:
Don't telly anyone :laughing: https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/components/views/rooms/MessageComposerButtons.tsx#L54